rspirv/lib.rs

```diff
@@ -1,5 +1,3 @@
-#![doc(html_root_url = "https://docs.rs/rspirv/0.7/")]
```

Comment:
I don't think this is still needed for docs.rs? This is the first time I'm seeing it...

Reply:
It's not something I've seen other crates do either.